linera-base/src/data_types.rs

```diff
@@ -284,6 +284,8 @@ pub struct Resources {
     pub message_size: u32,
     /// An increase in the amount of storage space.
     pub storage_size_delta: u32,
+    /// A number of service as oracle requests to be performed.
```

Comment:
```suggestion
    /// A number of service-as-oracle requests to be performed.
```

It somehow feels better this way.